pcsx2/CDVD/CDVD.cpp

```diff
@@ -1369,17 +1368,21 @@ static uint cdvdStartSeek(uint newsector, CDVD_MODE_TYPE mode)
 	}
 
 	// Only do this on reads, the seek kind of accounts for this and then it reads the sectors after
-	if (delta && !isSeeking)
+	if ((delta || cdvd.Action == cdvdAction_Seek) && !isSeeking)
 	{
-		int rotationalLatency = cdvdRotationalLatency((CDVD_MODE_TYPE)cdvdIsDVD());
-		//DevCon.Warning("%s rotational latency at sector %d is %d cycles", (cdvd.SpindlCtrl & CDVD_SPINDLE_CAV) ? "CAV" : "CLV", cdvd.SeekToSector, rotationalLatency);
+		uint rotationalLatency = cdvdRotationalLatency((CDVD_MODE_TYPE)cdvdIsDVD());
```

Comment:
Can be const and remove the empty row below.

Reply:
Actually, looks like I accidentally removed the logging, that's why there's a gap, but yeah will const.